assets/js/tabs.js

```diff
@@ -0,0 +1,24 @@
+$('.tab-content').find('.tab-pane').each(function (idx, item) {
```

Comment:
curious why we're mixing jquery dom traversal with browser APIs in e.g. try.js.  Is there a reason not to consistently use one or other?

Reply:
I have refactored the scripts. The style differs originally because I reused some scripts from an earlier project.